vignettes/DBI.Rmd

```diff
@@ -13,109 +13,183 @@ vignette: >
 knitr::opts_chunk$set(echo = TRUE)
 ```
 
-The {DBI} package defines a common interface between the R and database management systems (DBMS).
-Hence the name: DBI stands for **d**ata**b**ase **i**nterface.
+### Who this tutorial is for
+- You have an understanding of R
+- You want to access or manipulate data in a database that may be on your machine or on a different computer on the internet
```

Comment:
I think this second bullet point is the only really important item here. Can you link this sentence to the subsequent text?